app/styles/globals.css

```diff
@@ -91,8 +91,6 @@ thead {
 th,
 td {
   padding: 10px;
-  width: 140px;
-  display: table-cell;
```

Comment:
This might affect some tables in the bdb. Worth a check

Reply:
Has this been investigated?

Reply:
Does not look like it changes anything